veinminer-bukkit/src/main/java/wtf/choco/veinminer/listener/BreakBlockListener.java

```diff
@@ -50,6 +44,10 @@
 import wtf.choco.veinminer.util.VMConstants;
 import wtf.choco.veinminer.util.VMEventFactory;
 
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
```

Comment:
These should stay at the top please. Alphabetical import order. I think there's a Checkstyle rule for this too.